### PR TITLE
[New Sample] Azure Open AI based sample "Enterprise Search App"

### PR DESCRIPTION
**Pull request by ChetanSharma-msft**
Open AI based sample with below capabilities:

1) Bot File Upload to Blob
2) Open AI Embeddings
3) Redis Search
4) Open AI Completion API

### CODE REVIEW

File: samples/bot-ai-enterprise-search/nodejs/README.md [hunk 1]
```diff
@@ -0,0 +1,240 @@
+---
+page_type: sample
```
Comment (ChetanSharma-msft) [inline samples/bot-ai-enterprise-search/nodejs/README.md:2]: Need to add build pipeline and Link to root readme file.

**Review by ChetanSharma-msft — COMMENTED** [review]
Added review comment.

File: samples/bot-ai-enterprise-search/nodejs/prompts/chat/skprompt.txt [hunk 2]
```diff
@@ -0,0 +1,4 @@
+The following is a conversation with an AI assistant, its name is Teams Chef.
```
Comment (ChetanSharma-msft) [inline samples/bot-ai-enterprise-search/nodejs/prompts/chat/skprompt.txt:1]: Need to check and remove if it is not needed.

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5710770366

**Review by aratigurav — COMMENTED** [review]
Please update the title to reflect Azure Open AI based sample. I can see some of the meeting app related information which is not relevant. In the deployment or how to run application mention about the valid domain to be added.

File: samples/bot-ai-enterprise-search/nodejs/prompts/chat/skprompt.txt (hunk 2 above)
Comment (ChetanSharma-msft) [reply samples/bot-ai-enterprise-search/nodejs/prompts/chat/skprompt.txt]: Removed it.

File: samples/bot-ai-enterprise-search/nodejs/README.md (hunk 1 above)
Comment (ChetanSharma-msft) [reply samples/bot-ai-enterprise-search/nodejs/README.md:2]: Added build pipeline and root readme links.

**Comment by ChetanSharma-msft** [comment]
> Please update the title to reflect Azure Open AI based sample. I can see some of the meeting app related information which is not relevant. In the deployment or how to run application mention about the valid domain to be added.

1) Updated Title and other docs.
2) Removed meeting related fix and raised another PR for that.
3) Added step to update valid domain and configuration for web app.

**Comment by aratigurav** [comment]
@ChetanSharma-msft couple of observations: 

1. Solution architecture diagram is not visible on readme page.
2. Interaction with bot -admin section is empty.
3. Interaction with bot- user section is empty.

**Review by aratigurav — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
Executed build pipeline: https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/5729965655

**Review by Mohammed-MSFT — APPROVED** [review]
Changes Looks Fine Approving It.

**Review by Wajeed-msft — APPROVED** [review]

**Review by Prithvi-MSFT — APPROVED** [review]

**Comment by ChetanSharma-msft** [comment]
> @ChetanSharma-msft couple of observations:
> 
> 1. Solution architecture diagram is not visible on readme page.
> 2. Interaction with bot -admin section is empty.
> 3. Interaction with bot- user section is empty.



> @ChetanSharma-msft couple of observations:
> 
> 1. Solution architecture diagram is not visible on readme page.
> 2. Interaction with bot -admin section is empty.
> 3. Interaction with bot- user section is empty.

Fixed the images and gif images.

**Pull request merged**